app/css/common/typography.css

```diff
@@ -0,0 +1,35 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat:400,700);
+
+@value montserrat: 'Montserrat', Helvetica, Arial, sans-serif;
+
+.heading {
+  font-family: montserrat;
+  font-size: 28px;
+  font-weight: 700;
+
+  line-height: 48px;
+}
+
+.subheading {
+  font-family: montserrat;
+  font-size: 21px;
+  font-weight: 700;
+
+  line-height: 24px;
+}
+
+.normal {
+  font-family: montserrat;
+  font-size: 16px;
+  font-weight: 400;
+
+  line-height: 24px;
```

Comment:
I wanted to try the vertical rhythm based off [this article](http://zellwk.com/blog/why-vertical-rhythms/), setting it explicitly to 24px